src/lib.rs

```diff
@@ -114,7 +115,7 @@ mod tests {
 
         fn write<T, I>(&mut self, iterator: T) -> Result<(), Self::Error>
         where
-            T: Iterator<Item = I>,
+            T: IntoIterator<Item = I>,
```

Comment:
Yup - this fixes a warning.  I had this locally, not sure why it didn't make it into the MR.